pkg/datastore/datastore_with_txcontext_test.go

```diff
@@ -41,22 +41,27 @@ func rollbackTx(t *testing.T, tx *gorm.DB) {
 	}
 }
 
-func testTxCrud(t *testing.T, ds datastore.DataStore, ctx context.Context, myCokeApp *App, user1, user2 *AppUser) {
+func testTxCrud(t *testing.T, ds datastore.DataStore, ctx context.Context, user1, user2 *AppUser) {
 	t.Helper()
 	assert := assert.New(t)
 	var err error
 
 	txFetcher := authorizer.SimpleTransactionFetcher{}
 
-	// Querying of previously inserted records should succeed
+	t.Log("Querying created records in single transaction")
+	tx, err := ds.Helper().GetDBTransaction(ctx, datastore.GetTableName(user1), user1)
```

Comment:
Please use ds.GetTransaction() to init transaction. ds.Helper().GetDBTransaction() (or ds.GetDBTransaction()) is used in every CRUD function to either extract the transaction pointer out of ctx, or init a brand new one

Reply:
Done